spec.bs

```diff
@@ -25,7 +25,7 @@ spec: URL; urlPrefix: https://url.spec.whatwg.org/
 
 <h2 id=urlpattern-class>The {{URLPattern}} class</h2>
```

Comment:
The sectioning here has now become awkward since most of the section titled "The URLPattern class" is no longer about the `URLPattern` class :).

My suggestion:

- The `URLPattern` class -> "URL patterns"
    - Introduce an "Introduction" subsection to contain the introductory text and examples
    - Introduce a "The `URLPattern` class" subsection to contain the IDL and method definitions
    - Introduce "The URL pattern struct" subsection to contain the struct definition
    - Introduce "High-level operations" to contain "create", "match", "has regexp groups"
    - Everything else can continue living in "Internals" and "Constructor string parsing"
- (Simpler version: "Internals" can continue to contain most things, including my "The URL pattern struct" and "High-level operations" sections. But "Introduction" and "The `URLPattern` class" should be separate.)
- Rename "Patterns" to "Pattern strings" to be slightly less confusing as a sibling of "URL patterns"

The editors should of course feel free to override my suggestion here, as this very much a question of editorial choice.

Reply:
I feel the suggestions reasonable.  I have updated the specification with the complexed version.
It actually moved explanations, and made [=create=] exported.